gitops/azure-devops/ManifestGeneration.md

```diff
@@ -83,18 +88,35 @@ In Azure DevOps:
   ![SELECT REPO TYPE](images/select-git-repo.png)
 
 6. Choose your high level definition repo:
-    1. Azure DevOps Repo Example: ![SELECT REPO NAME](images/select-ado-repo.png)
-    1. GitHub Repo Example:![SELECT REPO NAME](images/select-github-repo.png)
+    1. Azure DevOps Repo Example: 
+
+    ![SELECT REPO NAME](images/select-ado-repo.png)
+    
+    1. GitHub Repo Example:
+
+    ![SELECT REPO NAME](images/select-github-repo.png)
 
-7. Select the "Configuration as Code" template
+7. If Azure DevOps detects your azure-pipelines.yaml, then skip to step #9. Otherwise, select the "Existing Azure Pipelines YAML file" template
 
   ![SELECT REPO TYPE](images/configuration-as-code.png)
 
-8. Name the build and choose the `azure-pipeline.yml` file in the root of your high level definition repo.
+8. Choose the yaml file from your repository and click "Continue"
```

Comment:
The screenshot below shows the file stored under a specific directory, but in the docs [here](https://github.com/microsoft/bedrock/blob/730a94d31f4e7b4b13253ae190e9a67711fd7e6c/gitops/azure-devops/ManifestGeneration.md#add-azure-pipelines-build-yaml) we recommend them to store in root. This might be a little confusing maybe?